support/commands.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA ;
+ */
+
+import 'cypress-file-upload';
```

Comment:
@anshul119 How are you referencing this file in your local project? The reason I ask is because for plugins, we originally proposed to have in `cypress.json`:
```
"pluginsFile": "node_modules/@oat-sa/e2e-runner/plugins"
```
And then should we add this new file?
```
"pluginsFile": "node_modules/@oat-sa/e2e-runner/plugins",
"supportFile": "node_modules/@oat-sa/e2e-runner/support",
```
The problem with that is that we already have a local support/commands file. So maybe it's better if we define local files for both:
```
"pluginsFile": "cypress/plugins",
"supportFile": "cypress/support",
```
And import the `e2e-runner` files inside the local files?

If we'll do it this way, there is some documentation to update in this repo.

Reply:
on Proctoring we are importing e2e-runner files inde local files, like [here](https://github.com/oat-sa/tao-proctoring-fe/blob/develop/cypress/support/commands.js#L19), so to use `cypress-file-upload` we should just `import '@oat-sa/e2e-runner/support/commands.js';`. I would suggest renaming `commands.js` to a more specific name like `fileUpload.js`, same as [was done](https://github.com/oat-sa/e2e-runner/tree/develop/support) for other commands.

We can not point `pluginsFile` config to local `cypress/plugins` because we can not impor e2e-runner plugins in the same way we do with commands, at least with the way e2e-runner [plugins file](https://github.com/oat-sa/e2e-runner/blob/master/plugins/index.js) is built.

Reply:
@adriloma I agree with keeping the `e2e-runner` "plugin-commands" like `cypress-file-upload` in separate files like you suggested. It will allow consumers to pick only what they need, instead of importing all commands.

As far as using a local plugins file to import and run the central plugins file, it looks possible, but increases the complexity so let's avoid it.

Reply:
I can rename the file, I kept it as commands.js so that we could put all the general commands which could be of use to any consumer there. but if you all prefer it to be in a dedicated file, that is fine for me as well!

Reply:
It seems deleted now - mistake?

Reply:
Strangely, git thought that I deleted the file while I just renamed it